vformer/models/classification/perceiver_io.py

```diff
@@ -0,0 +1,83 @@
+import torch
+import torch.nn as nn
+
+from ...common import BaseClassificationModel
+from ...decoder import PerceiverIODecoder
+from ...encoder import PerceiverIOEncoder
+from ...utils import MODEL_REGISTRY
+
+
+@MODEL_REGISTRY.register()
+class PerceiverIO(nn.Module):
```

Comment:
I didn't use BaseClassificationModel because img_size and patch_size are not that relevant in this model (model takes in image as a flattened array of pixels

Reply:
I see; that's alright.